VideoWeb/VideoWeb.Common/ConferenceService.cs

```diff
@@ -53,7 +53,8 @@ public async Task<Conference> GetConference(Guid conferenceId, CancellationToken
     /// <param name="conferenceId"></param>
     /// <param name="cancellationToken"></param>
     /// <returns></returns>
-    public async Task<Conference> ForceGetConference(Guid conferenceId, CancellationToken cancellationToken = default)
+    public async Task<Conference>
```

Comment:
You can revert the change here since the method name has moved into the new line

Reply:
done